q2_deblur/plugin_setup.py

```diff
@@ -128,8 +122,37 @@ def denoise(demultiplexed_seqs: SingleLanePerSampleSingleEndFastqDirFmt,
         'jobs_to_start': qiime2.plugin.Int,
         'hashed_feature_ids': qiime2.plugin.Bool
     },
+    parameter_descriptions={
+        'pos_ref_fp': ("Positive (16S) filtering database. Keep all sequences "
+                       "aligning to this FASTA file"),
+        'neg_ref_fp': ("Negative (artifacts) filtering database. Keep all "
+                       "sequences aligning to this FASTA file"),
+        'mean_error': ("The mean per nucleotide error, used for original "
+                       "sequence estimate. If not passed typical illumina "
+                       "error rate is used"),
+        'error_dist': ("A comma separated list of error probabilities for "
+                       "each hamming distance. The length of the list "
+                       "determines the number of hamming distances taken into "
+                       "account."),
+        'indel_prob': ('Insertion/deletion (indel) probability (same for N '
+                       'indels)'),
+        'indel_max': "Maximal indel number",
+        'trim_length': "Sequence trim length",
+        'min_reads': ("In final biom table - keep only sequences appearing at "
```

Comment:
*Retain only features appearing at least `min_reads` times across all samples in the resulting feature table.*